accumulator/pollard_test.go

```diff
@@ -215,6 +226,52 @@ func fixedPollard(leaves int32) error {
 	return nil
 }
 
+func PollardSerializeDeserialize() error {
+	var p Pollard
+
+	// generate slice of leaf
+	leaves := make([]Leaf, 10)
+
+	for i := 0; i < len(leaves); i++ {
+		leaves[i].Hash[0] = uint8(i + 1)
+	}
+
+	// add leaves to pollard
+	p.add(leaves)
+
+	// performing serialization
+
+	old_byte, err := p.Serialize()
+
+	if err != nil {
+		return err
```

Comment:
After changing the function to `TestPollardSerializeDeserialize(t *testing.T)`, this should be changed to `t.Fatal(err)` since Golang testing function signature must be `func TestXXX(t *testing.T) {}`

`t.Error()` (or `t.Errorf()` to have formatting) for errors that aren't fatal and when the test can still go on to give more useful information for debugging. `t.Fatal()` for errors that force the test to be stopped.

Reply:
okay working on it